es-vue-base/src/lib-components/EsReview.vue

```diff
@@ -49,25 +61,108 @@
                 :length="commentLimit"
                 @click="$emit('showMore')" />
             <template v-else>
-                {{ updatedComment || comment }}
+                <template v-if="!commentLimit && updatedComment">
+                    <div data-testid="both-comments">
+                        <span class="d-inline-block font-weight-bolder">Updated review: </span>
+                        {{ updatedComment }}
+                        <div
+                            class="font-size-sm mt-2">
+                            <span class="d-none d-lg-inline-block">Updated on </span>
+                            {{ updatedDate }}
+                        </div>
+                        <div class="mt-2">
+                            <span class="d-inline-block font-weight-bolder">Previous review: </span>
+                            {{ comment }}
+                        </div>
+                    </div>
+                </template>
+                <template v-else>
+                    {{ updatedComment || comment }}
+                </template>
             </template>
         </div>
-        <small class="d-flex align-items-center text-gray-800 text-nowrap">
-            <span
-                class="name-holder d-inline-block text-truncate pr-1"
-                data-testid="subtext-test">
-                <span class="d-none d-lg-inline-block">Posted by</span>
-                {{ reviewerName }}
-            </span>
-            <span class="date-holder d-inline-block">on {{ formattedDate }}</span>
-        </small>
-        <small class="d-none d-lg-flex align-items-center text-gray-800 mt-2">
+        <div class="d-flex flex-nowrap">
+            <small class="d-flex align-items-center text-gray-800 text-nowrap details-holder">
+                <span
+                    class="name-holder d-inline-block text-truncate pr-1"
+                    data-testid="subtext-test">
+                    <span class="d-none d-lg-inline-block">{{ updatedComment && commentLimit ? "Updated by" : "Posted by" }}</span>
+                    {{ reviewerName }}
+                </span>
+                <span class="date-holder">on {{ updatedComment && commentLimit ? updatedDate : formattedDate }}</span>
```

Comment:
Long logic blocks feel bad, this could be a computed property.


```
{{ displayDate }}

displayDate(){
return updatedComment && commentLimit ? updatedDate : formattedDate;
}
```